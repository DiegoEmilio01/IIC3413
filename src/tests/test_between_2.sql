SELECT *
FROM T1 as x
WHERE x.a BETWEEN "test-record-0" AND "test-record-2"