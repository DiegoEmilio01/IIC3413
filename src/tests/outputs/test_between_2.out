x.a,x.b
"test-record-0",30
"test-record-1",31
"test-record-2",32

 got 3 results.
